### PR TITLE
Sip Jibri

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: src/main/java/org/jitsi/jicofo/recording/jibri/CommonJibriStuff.java [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+     */
+    @Override
+    synchronized public void processPacket(Packet packet)
+    {
+        // FIXME uncomment
+        //if (logger.isDebugEnabled())
+        //{
+            logger.info("Processing an IQ: " + packet.toXML());
+        //}
+
```
Comment (damencho) [inline src/main/java/org/jitsi/jicofo/recording/jibri/CommonJibriStuff.java]: Maybe we should return those prints to debug as the commented code used to be.

File: src/main/java/org/jitsi/jicofo/recording/jibri/CommonJibriStuff.java [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+                                        final XmppChatMember    sender)
+    {
+        String senderMucJid = sender.getContactAddress();
+        // FIXME uncomment
+        //if (logger.isDebugEnabled())
+        //{
+        logger.info(
+            "Jibri request from " + senderMucJid + " iq: " + iq.toXML());
+        //}
+
```
Comment (damencho) [inline src/main/java/org/jitsi/jicofo/recording/jibri/CommonJibriStuff.java]: Debug?

File: src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java [hunk 3]
```diff
@@ -0,0 +1,756 @@
...
+        boolean accept = packet instanceof IQ
+            && currentJibriJid != null
+            && (packet.getFrom().equals(currentJibriJid) ||
+                    (packet.getFrom() + "/").startsWith(currentJibriJid));
+        //if (!accept)
+        //{
+          //  logger.info("SESSION REJECTING: " + packet.toXML());
+            //logger.info("CURRENT JIBRI: " + currentJibriJid +" from: " + packet.getFrom());
+        //}
+
```
Comment (damencho) [inline src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java]: FIXME cleanup

File: src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java [hunk 4]
```diff
@@ -0,0 +1,756 @@
...
+     */
+    @Override
+    synchronized public void processPacket(Packet packet)
+    {
+        //if (logger.isDebugEnabled())
+        //{
+            logger.info(
+                "Processing an IQ from Jibri: " + packet.toXML());
+        //}
+
```
Comment (damencho) [inline src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java]: Debug.

**Comment by paweldomas** [comment]
Oh I completely forgot about this FIXMEs. Yes, I will have that cleaned up.

**Pull request merged**